engine/src/main/java/com/datatorrent/stram/api/StreamingContainerUmbilicalProtocol.java

```diff
@@ -412,8 +412,10 @@ public String toString()
    * @param containerId
    * @param operators
    * @param msg
+   * @param logFileName
+   * @param logFileOffset
    */
-  void reportError(String containerId, int[] operators, String msg);
+  void reportError(String containerId, int[] operators, String string, String logFileName, long logFileOffset);
```

Comment:
msg, name and offset should be encapsulated into object. name and offset do not exist without the msg.